### ISSUE TITLE
Error attempting to `yarn prisma db seed`

### ISSUE THREAD

**Issue by tmeuze** [AlphaNecron/Chakram#21]
Hi @AlphaNecron , I'm encountering an error when attempting this (docker-compose). Any ideas?
```
/void # export DATABASE_URL=postgres://postgres:<password>@postgres/postgres
/void # yarn prisma db seed
yarn run v1.22.15
$ /void/node_modules/.bin/prisma db seed
Running seed command `ts-node --compiler-options {"module":"commonjs"} prisma/seed.ts` ...
PrismaClientKnownRequestError: 
Invalid `prisma.user.aggregate()` invocation:

  The table `public.User` does not exist in the current database.
    at cb (/void/node_modules/@prisma/client/runtime/index.js:38537:17)
    at async main (/void/prisma/seed.ts:8:17) {
  code: 'P2021',
  clientVersion: '3.3.0',
  meta: { table: 'public.User' }
}

An error occured while running the seed command:
Error: Command failed with exit code 1: ts-node --compiler-options {"module":"commonjs"} prisma/seed.ts
```
I've also tried `npx...` with the same result. 

To further elaborate: postgres container is running. I see the following errors in the container:
```
2022-02-15 15:14:17.132 UTC [1019] ERROR:  relation "public.User" does not exist at character 56
2022-02-15 15:19:43.828 UTC [1297] STATEMENT:  SELECT COUNT(*) FROM (SELECT "public"."User"."id" FROM "public"."User" WHERE 1=1 OFFSET $1) AS "sub"
2022-02-15 15:14:17.132 UTC [1019] STATEMENT:  SELECT COUNT(*) FROM (SELECT "public"."User"."id" FROM "public"."User" WHERE 1=1 OFFSET $1) AS "sub"
2022-02-15 15:19:43.828 UTC [1297] ERROR:  relation "public.User" does not exist at character 56
```

Thank you,

_Originally posted by @tmeuze in https://github.com/AlphaNecron/Void/issues/16#issuecomment-1040408758_

**Comment by AlphaNecron** [comment]
Can you do `docker ps`?

**Comment by tmeuze** [comment]
Sure thing. Here are the relevant containers (I can provide full output if desired):
```
f18e20491339   ghcr.io/alphanecron/void/void:v0 "docker-entrypoint.s…"   56 minutes ago   Up 56 minutes 0.0.0.0:9400->9400/tcp, :::9400->9400/tcp  void_void_1
1d424b01eedc   postgres  "docker-entrypoint.s…"   56 minutes ago   Up 56 minutes (healthy)   5432/tcp void_postgres_1
```
Edit: I have set `9400` as the [core] port in `config.toml`

Thank you for your time and attention.

**Comment by AlphaNecron** [comment]
Can you upload the logs of `void_postgres_1`? Have you tried restarting both containers?

**Comment by AlphaNecron** [comment]
Try `export DATABASE_URL=postgres://postgres:postgres@postgres/postgres`?

**Comment by tmeuze** [comment]
I did overlook the `database_url` config.toml parameter. I have since specified the password in the URL: `postgres://postgres:postgres@postgres/postgres` -> `postgres://postgres:<password>@postgres/postgres`

(is that correct?)

Now I'm getting authentication failures where I didn't before:
```
PostgreSQL Database directory appears to contain a database; Skipping initialization
2022-02-15 16:08:31.201 UTC [1] LOG:  starting PostgreSQL 14.2 (Debian 14.2-1.pgdg110+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 10.2.1-6) 10.2.1 20210110, 64-bit
2022-02-15 16:08:31.201 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
2022-02-15 16:08:31.201 UTC [1] LOG:  listening on IPv6 address "::", port 5432
2022-02-15 16:08:31.208 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
2022-02-15 16:08:31.213 UTC [26] LOG:  database system was interrupted; last known up at 2022-02-15 16:07:07 UTC
2022-02-15 16:08:31.381 UTC [26] LOG:  database system was not properly shut down; automatic recovery in progress
2022-02-15 16:08:31.383 UTC [26] LOG:  redo starts at 0/16FB188
2022-02-15 16:08:31.383 UTC [26] LOG:  invalid record length at 0/16FB270: wanted 24, got 0
2022-02-15 16:08:31.383 UTC [26] LOG:  redo done at 0/16FB238 system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s
2022-02-15 16:08:31.401 UTC [1] LOG:  database system is ready to accept connections
2022-02-15 16:08:36.151 UTC [33] FATAL:  password authentication failed for user "postgres"
2022-02-15 16:08:36.151 UTC [33] DETAIL:  Connection matched pg_hba.conf line 100: "host all all all scram-sha-256"
2022-02-15 16:10:09.330 UTC [111] FATAL:  password authentication failed for user "postgres"
2022-02-15 16:10:09.330 UTC [111] DETAIL:  Connection matched pg_hba.conf line 100: "host all all all scram-sha-256"
...
```
Though I'm certain I set the same password in all parameters and in the docker-compose.yml

`export DATABASE_URL=DATABASE_URL=postgres://postgres:<password>@postgres/postgres` did not do the trick:
```
  error: Error validating datasource `db`: the URL must start with the protocol `postgresql://` or `postgres://`.
  -->  schema.prisma:3
   | 
 2 |   provider = "postgresql"
 3 |   url      = env("DATABASE_URL")
 ```
Was the doubled `DATABASE_URL=` intentional?

Surely I'm just doing something dumb or overlooking something simple.

**Comment by AlphaNecron** [comment]
Yes, sorry, the default password for postgres in docker is always `postgres`, as well as the username, host and database name.

**Comment by tmeuze** [comment]
All good! Ok, I set everything back to default `postgres` password. Rebuilt both containers - same authentication failure.

Gotta be human-error, or something on my end. I'll keep playing around with it. 

Thank you so much.

**Comment by AlphaNecron** [comment]
You can try removing postgres image from docker and pull it once again. Hope this workaround would solve your problem.

**Comment by AlphaNecron** [comment]
- Stop all related containers
- Do `docker rmi postgres -f`
- Rebuild using Docker compose
However, you can consider changing from `v0` to `dev` branch. (By editing `docker-compose.yml`)

**Comment by tmeuze** [comment]
Hey @AlphaNecron , sorry for the delay, and thank you for your help. 

I took the postgres container out of the equation, and just used my own postgres server. Doing this, I was able to run the command to create the default user. 

Might be noteworthy - using the postgres container - on attempting to log in, I would not get a "user does not exist" error. No error at all.  When I specified my own database server, the error would then appear until I ran the command to create the user. 

Thanks again!
